### PR TITLE
Add docker documentation to the tile-converter

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/tile-converter/docs/cli-reference/tile-converter.md [hunk 1]
```diff
@@ -53,6 +53,51 @@ Run it with the local web server from project directory:
 $ I3sLayerPath="./data/CairoLayer" DEBUG=i3s-server:* npx i3s-server
 ```
 
+## Docker image
+
+Converter also available as docker image in the [visgl/tile-converter](https://hub.docker.com/r/visgl/tile-converter) repo.
+
+Run it to download tile-converter docker image:
```
Comment (ibgreen) [inline modules/tile-converter/docs/cli-reference/tile-converter.md:60]:
```suggestion
To download tile-converter docker image, run:
```

File: modules/tile-converter/docs/cli-reference/tile-converter.md [hunk 2]
```diff
@@ -53,6 +53,51 @@ Run it with the local web server from project directory:
...
+  --token ... \
+  --tileset ... \
+  --name ... \
+  --output ... \
+  --max-depth ...
+```
+
+Docker run arguments:
+
+```bash
```
Comment (ibgreen) [inline modules/tile-converter/docs/cli-reference/tile-converter.md:83]: Not sure this section should be `bash` formatted

**Review by ibgreen — APPROVED** [review]

File: modules/tile-converter/docs/cli-reference/tile-converter.md (hunk 2 above)
Comment (dryabinin-actionengine) [reply modules/tile-converter/docs/cli-reference/tile-converter.md]: Removed bash section

File: modules/tile-converter/docs/cli-reference/tile-converter.md (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/tile-converter/docs/cli-reference/tile-converter.md]: FIxed

**Pull request merged**